src/apps/lwaftr/icmp.lua

```diff
@@ -93,6 +94,22 @@ function new_icmpv4_packet(from_eth, to_eth, from_ip, to_ip, initial_pkt, l2_siz
    return new_pkt
 end
 
+function is_icmpv4(l2_size, pkt)
+   assert(l2_size > 2)
```

Comment:
Fixed, I have removed all the `l2_size` thingies, and now the apps assume a normal Ethernet header.